tka/sync.go

```diff
@@ -96,8 +96,8 @@ func (a *Authority) syncOffer() (SyncOffer, error) {
 // This SyncOffer + AUM exchange should be performed by both ends,
 // because its possible that either end has AUMs that the other needs
 // to find out about.
-func (a *Authority) SyncOffer() (SyncOffer, error) {
-	return a.syncOffer()
+func (a *Authority) SyncOffer(storage Chonk) (SyncOffer, error) {
+	return a.syncOffer(storage)
 }
```

Comment:
Export `a.syncOffer` and then this should be unnecessary?

Reply:
Done.